CONTRIBUTING.md

```diff
@@ -4,47 +4,47 @@ Please follow common guidelines for our projects [here](https://github.com/packi
 
 ## Reporting Bugs
 
-- [List of known issues](https://github.com/packit/packit-service/issues) and in case you need to create a new issue, you can do so [here](https://github.com/packit/packit-service/issues/new).
+- [List of known issues](https://github.com/packit/packit-service/issues); if you need to create a new issue, you can do so [here](https://github.com/packit/packit-service/issues/new).
 - Getting a version of `packit`:<br>
   `rpm -q packit` or `pip3 freeze | grep packitos`
 
 ## Building images locally
 
-For building images SOURCE_BRANCH build arg is required. It allows select which branch should be
-used during automated build and test process, based on deployment stage it was executed from.
+For building images the SOURCE_BRANCH build arg is required. It allows selecting which branch should be
+used during the automated build and test process, based on the deployment stage it was executed from.
 
-Docker file build arg SOURCE_BRANCH is set to:
+The Podman file build arg SOURCE_BRANCH is set to:
 
 - SOURCE_BRANCH env variable, which is available in custom build process([docs](https://docs.docker
   .com/docker-hub/builds/advanced/))
 - zuul.branch ansible variable, which is provided in zuul-ci environment. ([docs](- [scripts](scripts/) - devops scripts used in multiple repositories
   ))
 
-When executing manually it is required to provide:
+When you are invoking 'make' manually, you must provide:
 
 - variable SOURCE_BRANCH for make targets:
 
 e.g.
 `SOURCE_BRANCH=main`
 
-- arg `--build-arg SOURCE_BRANCH=value` when using docker(-compose) build
+- arg `--build-arg SOURCE_BRANCH=value` when using podman build
 
 e.g.
 
-    docker build --build-arg SOURCE_BRANCH=stable
-    docker-compose build --build-arg SOURCE_BRANCH=stable
+    podman build --build-arg SOURCE_BRANCH=stable
+    podman-compose build --build-arg SOURCE_BRANCH=stable
 
 If SOURCE_BRANCH is empty build will fail.
 If SOURCE_BRANCH is not empty and is not main or stable than main value will be used.
 
 ## Running packit-service locally
 
 Since packit-service is already a fairly complex system, it's not trivial to
-run it locally. This is why we are running everything in containers.
+run it locally. This is why we run everything in containers.
 
 This repository contains [docker-compose.yml](./docker-compose.yml) for
-[docker-compose](https://github.com/docker/compose). Before you run it, we
-suggest you to open the file and read all the comments.
+[podman-compose](https://github.com/containers/podman-compose). Before you run it, we
```

Comment:
Not sure this is a good idea.
We investigated `podman-compose` about a year ago in #703 and it was not a good replacement for `docker-compose` - and there have been no new [releases](https://pypi.org/project/podman-compose/#history) since then.
I've just tried the version packaged in F34 and see some errors, so if you want to play with that please do so in a separate PR. Thanks.

Reply:
Jiri,

Thanks for taking a look.  I think we could say that podman-vs.-docker issues are
somewhat outside the scope of making 'make check' more consistent, so I'm
putting the original text back.